opentreemap/treemap/js/src/base.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
```

Comment:
Consider a comment on this file to explain what it's for. Does this define what's in the common bundle?